issue found on: mock.go:70:2: importShadow

an issue has been detected in source code: ./mock.go:70:2: importShadow: shadow of imported from 'github.com/RedHatInsights/insights-results-aggregator-mock/groups' package 'groups'

--- a/mock.go
+++ b/mock.go
@@ -67,7 +67,7 @@ func startService(config conf.ConfigStruct) int {
 	groupsCfg := conf.GetGroupsConfiguration()
 	contentCfg := conf.GetContentConfiguration()
 
-	groups, err := groups.ParseGroupConfigFile(groupsCfg.ConfigPath)
+	ruleGroups, err := groups.ParseGroupConfigFile(groupsCfg.ConfigPath)
 	if err != nil {
 		log.Error().Err(err).Msg("Groups init error")
 		return ExitStatusServerError
@@ -86,7 +86,7 @@ func startService(config conf.ConfigStruct) int {
 		return ExitStatusServerError
 	}
 
-	serverInstance = server.New(serverCfg, storageInstance, groups, ruleContent)
+	serverInstance = server.New(serverCfg, storageInstance, ruleGroups, ruleContent)
 
 	err = serverInstance.Start()
 	if err != nil {

